string command
uint16 red
uint16 white
---
string response